Add error in troubleshooting page (#38723)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

| `401 Unauthorized`                           | [API and Browser Tests → Unauthorized](#api-and-browser-tests) |
| `Element not found`                            | [Test execution → Element selection issues](#element-detection-warning-in-browser-test-steps) |
| `Unsupported browser version`                  | [Private Locations → Browser compatibility](?tab=common#requirements-for-browser-tests-running-on-private-location) |

> | `Failed to load resource: ERR_INVALID_ARGUMENT`| [Private Locations → Chrome 149 issue](?tab=common#chrome-149-issue-on-private-locations) |

Reviewer: ```suggestion
| `Failed to load resource: ERR_INVALID_ARGUMENT` | [Private Locations → Chrome 149 issue](?tab=common#chrome-149-issue-on-private-locations) |
```

{{< img src="synthetics/timeout.png" alt="API test on private location timing out" style="width:70%;" >}}

### Chrome 149 issue on private locations

> Chromium version 149 introduced stricter validation for request headers, which can cause sub‑resource requests to fail with `net::ERR_INVALID_ARGUMENT`. Earlier Chrome versions were more permissive.

Reviewer: ```suggestion
Chromium version 149 introduced stricter validation for request headers, which can cause subresource requests to fail with `net::ERR_INVALID_ARGUMENT`. Earlier Chrome versions were more permissive.
```

Reviewer: Also, you're mostly using "Chrome" throughout, but here you start with "Chromium." I would stick to one, or make the relationship between them clear.

> **Solution:** Upgrade your private location to [version 1.69.1 or later][18] to resolve this issue.

Reviewer: ```suggestion
**Solution**: Upgrade your private location to [version 1.69.1 or later][18] to resolve this issue.
```

Style nit

[14]: /synthetics/guide/step-duration
[16]: /synthetics/network_path_tests/#agent-configuration
[17]: /network_monitoring/network_path/setup/?tab=linux#increase-the-number-of-workers

> [18]: https://hub.docker.com/r/datadog/synthetics-private-location-worker

Reviewer: Because of how tabs work, you have to put link references within the same tab they're referenced in. Otherwise they'll render broken like this:

<img width="647" height="45" alt="Image" src="https://github.com/user-attachments/assets/50761630-760f-4987-9cc1-fc4d1b640440" />

Could you please move this within the Common tab as `[106]`?
